pages/manage/productions/create.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="wrapper">
     <section class="container-fluid productions-create-page">
-      <h1 class="title create-title">{{ $t('productions.create') }}</h1>
+      <pub-h1 :variant="heading1Variant.TITLE" class="create-title">{{
```

Comment:
I don't think we need a variant for this. So far the only place where it's not used is the start page, but the `h1` there is actually hidden. It's used as a replacement text for screen readers that do not see the UiTdatabank logo in the SVG. So the styling doesn't matter there. So I'd suggest to always add the `border-bottom` by default and not have a variant for that.

Reply:
See the style that hides the h1 on the start page here: https://github.com/cultuurnet/udb3-frontend/pull/47/files#diff-8df541b53e4cb5088c52e8d2c73e2b09L484

(We could even let that stay a normal `h1` since it's not visible anyway...)